### ISSUE TITLE
NAMEOF_ENUM compiled failed with MSVC 17.2.2

### ISSUE THREAD

**Issue by YunchengLiu** [Neargye/nameof#38]
Thank you for this excellent program that has helped me so much in my work!

I currently have a problem that after upgrading the MSVC version to 17.2.2, the previous code does not compile properly.

packages are configured via vcpkg and all up to date.

as follows.（compiled with -std=c++20）

```
#include <nameof.hpp>
#include <fmt/format.h>

enum class ObjectiveLensMedia {
    Air = 0,
    Oil = 1
};

int main() {
    fmt::print("{}.\n", NAMEOF_ENUM(ObjectiveLensMedia::Air));
}
```

Will give me a static assert failed with message '**nameof::nameof_enum requires enum implementation and valid max and min.**'.

I'm very confused about this and would like help, a lot of my code depends on this.

Thank you very much!

**Comment by Neargye** [comment]
Duplicate https://github.com/Neargye/nameof/issues/34

**Comment by Neargye** [comment]
Will be included in v0.10.2, or use master if need fixes now.

**Comment by YunchengLiu** [comment]
Thanks so much for such a timely response! I will wait for the RELEASE!
